stub/stub.meta

```diff
@@ -0,0 +1,45 @@
+[ccpp-table-properties]
+  name = stub
+  type = scheme
+  dependencies =
+
+########################################################################
+[ccpp-arg-table]
+  name = stub_init
+  type = scheme
+[errmsg]
+  standard_name = ccpp_error_message
+  long_name = error message for error handling in CCPP
+  units = none
+  dimensions = ()
+  type = character
+  kind = len=*
+  intent = out
+[errflg]
+  standard_name = ccpp_error_flag
```

Comment:
Didn't this get changed to `ccpp_error_code`?

Reply:
Yes, Ufuk's code is currently behind NCAR ccpp-{framework,physics}) main. I will need to do the update when Ufuk and I update our various submodule branches with the latest codes (we need to do that at the same time).

Reply:
Is that before or after this gets merged? If it is in transition, perhaps it is too early to bring it to main?

Reply:
Before. I should make this PR a draft PR for now.

Reply:
Code is now up to date and ready for review and merging.